sceptre/cli/update.py

```diff
@@ -67,8 +68,11 @@ def update_command(ctx, path, change_set, verbose, yes):
             for description in list(descriptions.values()):
                 if not verbose:
                     description = simplify_change_set_description(description)
-                write(description, context.output_format)
 
+                write('-'*80,'text')
+                write(yaml.dump(description), 'text')
```

Comment:
Why is this hard-coded to yaml format instead of using the output_format that has been explicitly requested by the user? Sceptre supports 3 output format. What am I missing here?

Reply:
Per #1180, the `write()` function does not appear to respect the `output_format`. I posted this patch to the issue and @zaro0508 requested a PR.

Note that this PR will need a new commit (5b4c2c3) which adds the missing yaml library dependency.

Reply:
As an aside, given that this is an explicitly interactive process, I'd argue that the more readable approach should always be used since we don't anticipate the output being piped to another process.

Reply:
@alexharv074 Is this, by any chance, the same issue you pointed out regarding the `write()` function? Or is this different?

Reply:
@dboitnot, rather than doing this to bypass the output format on `write`, I'd rather you _fix_ the `write` function. That seems like the more appropriate fix. I don't think this is the first time the weaknesses with `write` have come up recently. But by providing a global option for output format, Sceptre makes "contract" with the user. I don't want to override that to force yaml (despite the superiority of yaml over json, IMO).

Reply:
@jfalkenstein, as mentioned above, this PR was created by request from a workaround I posted to #1180. The original issue actually reports this as an error with `write()`.

I agree 100% with keeping our promise to the user about output formatting. I do wonder if that applies in this case because it's an explicitly interactive process. But then it's possible I don't fully grasp the purpose of the `--output` flag or the `write()` function.

Reply:
What do you mean by "explicitly interactive"? I'm confused about that bit.

Reply:
Fixing the output format and what the default output format should be are two separate issues.  Let's focus on fixing the output format first.   I agree with @jfalkenstein we should either fix the write function or replace it with a library that can do for us.

Reply:
@jfalkenstein, Sorry, by "explicitly interactive" I mean that that the `update -c` command presents user-readable output and pauses for user interaction (pressing 'y' or 'n'). To me this suggests we're assuming a user is interacting with the command at a terminal and that we're not running in a pipe or script which might parse the output. It's for that reason I wonder if the `--output` contract applies in this case.

@zaro0508, This PR is just the workaround I posted to #1180 and I have no objection to discarding it in favor of addressing the issue with `write()`.

Reply:
@dboitnot , sorry I lost track of this convo with the holidays and such. #1169 is soon to be merged, which fixes the issue with dumping yaml dicts. I suspect that once that is merged, this bug likely will go away. Not sure if this PR will even be necessary. 

With that said, there has been ongoing conversation around overhauling the `write()` function, as it has lots of inconsistencies and other issues with it and how it operates (especially around the `text` output type). There will likely be an issue around that soon.